recipes/databind.core/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "databind.core" %}
+{% set version = "1.1.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/databind.core-{{ version }}.tar.gz
+  sha256: a177dd96e954ee677ce2de80dce6112b8a5422a4ceb345f2e534f86768e1b66c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
+  run:
+    - deprecated >=1.2.12,<2.0.0
+    - nr.optional >=0.1.1,<1.0.0
+    - nr.parsing.date >=1.0.1,<2.0.0
+    - nr.preconditions >=0.0.4,<1.0.0
+    - nr.stream >=0.1.1,<1.0.0
+    - python >=3.6,<4.0
+    - typing-extensions <4.0.0,>=3.10.0
+    - typing_extensions >=3.10.0,<4.0.0
```

Comment:
`typing-extensions` depends on `typing_extensions` (and only that) so you really only need to depend on one of these`